needls/server.py

```diff
@@ -510,6 +512,23 @@ def update_settings(ls, *args):
         )
         return
 
+    if ext_conf_py_file:
```

Comment:
See above.
I don't think that we need an extra handling.

This kind of error-checks should be done always. Even if it is `conf.py`.